backend/messaging/wizard_topic.go

```diff
@@ -1,41 +1,92 @@
 package messaging
 
-import "sync"
+import (
+	"context"
+	"errors"
+	"strconv"
+	"sync"
+
+	"github.com/sensu/sensu-go/types"
+)
 
 // WizardTopic encapsulates state around a WizardBus topic and its
 // consumer channel bindings.
 type WizardTopic struct {
+	ID       string
```

Comment:
ID, Bindings and Ring are all implementation details and shouldn't be exported.

Reply:
Yeah, they don't need to be, but for that matter, neither does WizardTopic.

Reply:
It's definitely worth reducing the API surface as much as possible, so if WizardTopic can be unexported, I think we should do that.